styles/sections/about.css

```
@@ -0,0 +1,197 @@
+@media (max-width: 767px) {
+
+  .about {
+    display: flex;
+    flex-direction: column;
+
+    padding: 16px 10px 16px 10px;
+  }
+
+  .about h2 {
+    text-align: left;
+    letter-spacing: 1.6px;
+    text-transform: uppercase;
+
+    font-size: 54px;
+    font-weight: 900;
+    line-height: 54px;
+  }
+
+  .about p {
+    margin: 16px auto 0px;
+
+    text-align: left;
+    letter-spacing: 1px;
+
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+  }
+
+  .img_about {
+    width: 100%;
+    margin: 16px auto 16px;
+
+  }
+
+  .img-block {
+    order: +1;
```

```suggestion
    order: 1;
```

А что тут дает плюс?